### ISSUE TITLE
adblock detected on this site

### ISSUE THREAD

**Issue by dylantrevor87** [brave/brave-browser#23550]
## Description 

on this site the brave adblocking is detected:
https://maxstream.video/flw44cqfpwsm/WHBWU1l0eFoyQVRaRGlFelI5WjBHZz09

## Steps to Reproduce

   1.  just load the page with adblock enabled
   2. 
   3. 

## Version/Channel Information:

- Can you reproduce this issue with the current release? 
- Can you reproduce this issue with the beta channel? 
- Can you reproduce this issue with the nightly channel? 

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards? 
- Is the issue reproducible on the latest version of Chrome?

**Comment by ryanbr** [comment]
Unable to load site @dylantrevor87  Is there an updated url?

**Comment by ryanbr** [comment]
Okay the message is shown due to `maxstream.video##+js(ra, style, div[style="display: none;"])` It should be working? cc: @antonok-edm 

Related uBO filters:
```
filters/filters-2021.txt:maxstream.video##+js(ra, style, div[style="display: none;"])
filters/filters-2021.txt:maxstream.video##+js(nostif, height)
filters/filters-2021.txt:maxstream.video##+js(nowoif)
filters/filters-2021.txt:@@||maxstream.video^$script,1p
filters/filters-2021.txt:@@||maxstream.video^$ghide
filters/filters-2021.txt:maxstream.video##h1:has-text(/adblock|supporter/) 
```

**Comment by dylantrevor87** [comment]
Any news on this?
Currently I have added lots of custom block lists but I still see ads.

**Comment by ryanbr** [comment]
Is there an updated url (the current reported link is dead) @dylantrevor87

**Comment by dylantrevor87** [comment]
try these:

https://maxstream.video/ivqjqov9jwjk/L3ZGOFo4QW9YUy83ZTNReEpkZ3Rydz09
https://maxstream.video/ivqjqov9jwjk/VkdFOWxnbnhlUjVMZWpTT1pMUEc0dz09
https://maxstream.video/icv6i9seqyax/VkdFOWxnbnhlUjVMZWpTT1pMUEc0dz09

(the ads don't show always, if they don't show up try reloading, or clear the cookies).

I tried blocking some ads with the picker (which by the way on this site sometimes the picker gets stuck and I have to close it, and reopen it).

Some ids I have blocked:
maxstream.video###container-9aab413bb15cdf136a593ce56f76fbf449251
maxstream.video###container-9aab413bb15cdf136a593ce56f76fbf438552
maxstream.video###container-9aab413bb15cdf136a593ce56f76fbf427077
maxstream.video###container-9aab413bb15cdf136a593ce56f76fbf427885
maxstream.video###container-9aab413bb15cdf136a593ce56f76fbf418775
maxstream.video###container-9aab413bb15cdf136a593ce56f76fbf465216
maxstream.video###container-9aab413bb15cdf136a593ce56f76fbf457399

But it seems useless: probably these ids are random generated.
These should be a different way to block these ads.

**Comment by ryanbr** [comment]
```
Access denied Error code 1020
You cannot access maxstream.video. Refresh the page or contact the site owner to request access.
```

possibilty hard links are being blocked, none of them worked

**Comment by dylantrevor87** [comment]
try this:
https://uprot.net/msfld/ffgpwh979cu9

**Comment by ryanbr** [comment]
Testing a fix, should land tomorrow.

**Comment by 424344** [comment]
thanks!

**Comment by ryanbr** [comment]
Should be fixed now @424344 can you confirm?

**Comment by dylantrevor87** [comment]
No, it's still not fixed. 
I'm seeing ads mainly in the lower right corner of the page, and sometimes in the center or top of the page.
I have taken a screenshot of the HTML (the ad is an iframe).
You can find the screen here:
https://ibb.co/y6tK3kX

**Comment by dylantrevor87** [comment]
Sorry, forget my previous comment.
While testing I had previously disabled Brave's adblocker, and forgot to enable it again.
If I enable it again the web site detects the adblocker, shows an alert saying to disable the adblocker, and then hides the video player.

**Comment by ryanbr** [comment]
It will still show the text, but the video plays fine here. (Clean profile, default shields)

https://gif.fanboy.co.nz/gif/maxstream.gif

**Comment by dylantrevor87** [comment]
please can you tell me what lists have you enabled?
and are you on stable release? I'm using v1.43.93.

I have just renamed my current brave profile, so on startup it was created a new one.
Did absolutely no changes to settings.

I loaded the uprot.net page, but had to disable shields on it, because the "continue" button wasn't shown.
this is strange because in your video you have shields enabled on uprot, and it works...

then I loaded maxstream, the player appears, but after some seconds the alert pops up (saying "disable your adblock please"), and it blocks the brave ui.
even if I don't click on the alert the player stops after some minutes.
and when I click it the player is hidden.

**Comment by ryanbr** [comment]
Only Easylist Cookie enabled as an extra list, this has no affect on the site however. Try manually updating Brave Ad blocker in brave://components/   Ensure shields is up. Test in private window mode also (no extensions).  
Video still plays here

**Comment by 424344** [comment]
I have just installed Brave beta, without changing options, but doesn't work like stable release.
I manually updated components. All is updated.
I also checked the adblock lists, and I have found enabled only "Easy List Italy".
Is it correct? 
Only one list enabled by default?

**Comment by 424344** [comment]
found ths issue: I have preferred language set to italian, then english.

![image](https://user-images.githubusercontent.com/26486758/193147447-f3e9d9b9-cb51-4bcd-b9c9-0eca5731a943.png)

in this configuration the brave adblocker doesn't work, and the popup is showed after some seconds.

If I swap the languages, putting english at the top then the brave adblocker works, and the popup is not shown.

**Comment by 424344** [comment]
any news on fixing the issue without changing language?
